### ISSUE TITLE
Add CONTRIBUTING.md

### ISSUE THREAD

**Issue by kartikey369-ind** [AmateursLeague/sneaky-package#38]
I Want to add contribution.md file to the repository also add hacktoberfest and gssoc-extd lable if possible

**Comment by Brijeshthummar02** [comment]
i can work on this, assign me the task.
Could you please add the Hacktoberfest label to my contributions, as I am participating in this event?

**Comment by jindalpriyanshu101** [comment]
Assigned @kartikey369-ind.

**Comment by this-is-yaash** [comment]
Okay, create a file named `CONTRIBUTIONS.md` in the root directory. Make sure to link to `CONTRIBUTIONS.md` in the `README.md` file.

Please fork and use this repository. Check how the project works and how the operations are executed!

Then, create the contributions documentation. 

If you want to contribute to this issue, please follow these three steps:
1. Fork the repository and create your own branch to work on the issue.
2. Avoid making changes in the `main` branch.
3. Ensure that you use valid commit messages.


In simple terms, conduct thorough research while documenting, as it plays a vital role in onboarding new contributors.

**Comment by jindalpriyanshu101** [comment]
> i can work on this, assign me the task. Could you please add the Hacktoberfest label to my contributions, as I am participating in this event?

Sorry buddy, we're following first come first server, you should be creator of the issue to get it assigned, OR should be first one to comment "Assign it to me" if the issue is created by maintainer.

**Comment by kartikey369-ind** [comment]
done with my work https://github.com/this-is-yaash/sneaky-package/pull/41
